### ISSUE TITLE
Mac os sequoia support

### ISSUE THREAD

**Issue by abd3lraouf** [ivoronin/TomatoBar#80]
This nice app is not working correctly on Mac OS sequoia. 
IT opens an empty window when the system restarts

**Comment by sinanbozkus** [comment]
same problem.

**Comment by GloryAlex** [comment]
same problem, it annoys.

**Comment by Aldo-cav** [comment]
> This nice app is not working correctly on Mac OS sequoia. IT opens an empty window when the system restarts

Yes, same issue here (Sequoia 15.1). BTW once you close that empty window the app is completely working. Maybe the developers are a little bit busy, because the last release it's more then 1 year old.

**Comment by sinanbozkus** [comment]
I use this one https://github.com/ivoronin/TomatoBar/issues/85.

**Comment by Aldo-cav** [comment]
> I use this one #85 to fix the all problems.

Oh, wonderful! Thank you.
